src/java.xml/share/classes/com/sun/org/apache/xerces/internal/impl/msg/XPointerMessages.properties

```diff
@@ -21,14 +21,14 @@
 
 # Messages for message reporting
 BadMessageKey = The error message corresponding to the message key can not be found.
-FormatFailed = An internal error occurred while formatting the following message:\n  
+FormatFailed = An internal error occurred while formatting the following message:\n
```

Comment:
Same here with `:\n`...

Reply:
It's done with the code (that is, a key is appended with the code). In fact, the whole Xerces stack was implemented this way. I'd leave it as is.